src/test/java/com/amazon/pvar/tspoc/merlin/CallGraphTests.java

```diff
@@ -54,15 +57,28 @@ public CallGraphTests(CallGraphTest cgt) {
     }
 
     private void runFindCalleeTest(FindCallees findCallees) {
-        org.apache.log4j.Logger.getRootLogger().setLevel(Level.OFF);
+        org.apache.log4j.Logger.getRootLogger().setLevel(Level.DEBUG);
         final var queryManager = new QueryManager();
         final var callNode = findCallees.callNode();
-        if (callNode.getFunctionRegister() == -1) {
-            throw new RuntimeException("not supported yet");
+        final Value calleeQueryValue;
+        final dk.brics.tajs.flowgraph.jsnodes.Node startingLocation;
+        if (callNode.getFunctionRegister() != -1){
+            calleeQueryValue = new Register(callNode.getFunctionRegister(), callNode.getBlock().getFunction());
+//            final var preds = FlowgraphUtils.predecessorsOf(callNode).toList();
+//            if (preds.size() != 1) {
+//                throw new RuntimeException("Expected exactly on predecessor of " + callNode);
+//            }
+//            startingLocation = preds.get(0);
```

Comment:
same comment as above. I prefer to delete commented out code if we're not going to use it. Either that or make a comment about why we're using it.

Reply:
Fixed in new revision